.githooks/README.md

```diff
@@ -0,0 +1,39 @@
+# Git Hooks
+
+## Overview
+
+This directory provides Git hooks that you can use in a standard Git development flow.
+
+## Installation
+
+To use Git hooks in your local Git repository, configure the default Git hooks path.
+To do so, run this command in the root of your `kyma-dashboard` Git repository:
+```shell
+git config core.hooksPath .githooks
+```
+Now, your local Git repository uses Git hooks from the `.githooks` path.
+
+## Troubleshooting
+
+### Mac: VCS Integration in GoLand Can't Use Applications Installed by Homebrew
+
+#### Cause
+
+This is a common issue for all JetBrains IDEs that run on a Mac.
+GoLand initializes the `PATH` upon startup based on the macOS `launchd` path.
+<!-- markdown-link-check-disable-next-line -->
+#### Remedy
+
+1. Add the `/usr/local/bin` path to the system `PATH` variable using the following command:
+   ```shell
+   sudo launchctl config user path /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
+   >**CAUTION:** This command introduces the change to ALL computer users.
```

Comment:
Please fix the formatting.

![Screenshot 2024-06-19 at 08 20 57](https://github.com/kyma-project/kyma-dashboard/assets/23300533/f3dcd179-a8d7-4814-93d7-07953157cfd4)

We need to end the code block first, leave an empty line, and only in the following one add the caution note.